src/textual/widgets/_footer.py

```diff
@@ -232,7 +232,6 @@ def compose(self) -> ComposeResult:
                 Footer.ctrl_to_caret,
                 Footer.compact,
             )
-        self.log(bindings)
```

Comment:
Think this was left accidentally from a recent update. It prints a LOT to the console 😄